packages/wdio-cli/src/launcher.js

```diff
@@ -276,12 +280,14 @@ class Launcher {
 
     /**
      * Close test runner process once all child processes have exited
-     * @param  {Number} cid  Capabilities ID
-     * @param  {Number} childProcessExitCode  exit code of child process
+     * @param  {Number} cid       Capabilities ID
+     * @param  {Number} exitCode  exit code of child process
      */
-    endHandler (cid, childProcessExitCode) {
-        this.exitCode = this.exitCode || childProcessExitCode
-        this.runnerFailed += childProcessExitCode !== 0 ? 1 : 0
+    endHandler ({ cid, exitCode }) {
+        const passed = exitCode === 0
+        this.exitCode = this.exitCode || exitCode
+        this.runnerFailed += !passed ? 1 : 0
```

Comment:
Should this always be 1 or 0? If so changing += to = makes more sense in my opinion

Reply:
nope, `runnerFailed` captures the amount of failing tests (important in line 143)